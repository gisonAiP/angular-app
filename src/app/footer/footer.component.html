<footer data-test="footer" class="d-flex d-md-none justify-content-around align-items-center py-2">
  <div class="text-center">
    <img src="./assets/svg/home.svg" width="22"/>
    <p class="m-0">Home</p>
  </div>
  <div class="text-center">
    <img src="./assets/svg/track.svg" width="22"/>
    <p class="m-0">Trip</p>
  </div>
  <div class="text-center">
    <img src="./assets/svg/compass.svg" width="22"/>
    <p class="m-0">Explore</p>
  </div>
  <div class="text-center">
    <img src="./assets/svg/repeat.svg" width="22"/>
    <p class="m-0">Marketplace</p>
  </div>
  <div class="text-center">
    <img src="./assets/svg/menu.svg" width="22"/>
    <p class="m-0">Menu</p>
  </div>
</footer>
