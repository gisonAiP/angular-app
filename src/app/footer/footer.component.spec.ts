import { createRoutingFactory, Spectator } from '@ngneat/spectator/jest';
import { FooterComponent } from './footer.component';

describe('FooterComponent', () => {
  let spectator: Spectator<FooterComponent>;
  const createComponent = createRoutingFactory({
    component: FooterComponent,
    shallow: true,
    providers: [],
    declarations: []
  });

  beforeEach(() => {
    spectator = createComponent();
    spectator.detectChanges();
  });

  it('should validate snapshot', () => {
    expect(spectator.fixture).toMatchSnapshot();
  });
});
