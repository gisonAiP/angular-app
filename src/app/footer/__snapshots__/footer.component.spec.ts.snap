// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent should validate snapshot 1`] = `
<app-footer>
  <footer
    class="d-flex d-md-none justify-content-around align-items-center py-2"
    data-test="footer"
  >
    <div
      class="text-center"
    >
      <img
        src="./assets/svg/home.svg"
        width="22"
      />
      <p
        class="m-0"
      >
        Home
      </p>
    </div>
    <div
      class="text-center"
    >
      <img
        src="./assets/svg/track.svg"
        width="22"
      />
      <p
        class="m-0"
      >
        Trip
      </p>
    </div>
    <div
      class="text-center"
    >
      <img
        src="./assets/svg/compass.svg"
        width="22"
      />
      <p
        class="m-0"
      >
        Explore
      </p>
    </div>
    <div
      class="text-center"
    >
      <img
        src="./assets/svg/repeat.svg"
        width="22"
      />
      <p
        class="m-0"
      >
        Marketplace
      </p>
    </div>
    <div
      class="text-center"
    >
      <img
        src="./assets/svg/menu.svg"
        width="22"
      />
      <p
        class="m-0"
      >
        Menu
      </p>
    </div>
  </footer>
</app-footer>
`;
